### PR TITLE
KL10 front end and other PDP-11 programs

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
Grab bag of various KL10 front end and PDP-11 tools and programs.

- KLFEDR, tool for KL10 front end filesystem
- FEDUMP, KL10 front end dumper
- 11STNK, PDP-11 linker
- KLRUG, PDP-11 debugger
- 11DDT, PDP-11 debugger
- KLDCP, KL10 diagnostics console program
- NTSDDT, KL10 exec DDT
- IOELEV, PDP-11 I/O front end
- TV-11, Knight TV PDP-11
- 11BOOT, KL10 front end booting
- STUFF, load PDP-11 through Rubin 10-11

### CODE REVIEW

File: build/misc.tcl [hunk 1]
```diff
@@ -895,6 +895,10 @@ respond "*" ":midas /t device;jobdev glp_sysen2;xgpdev\r"
 respond "with ^C" "GLP==1\r\003"
 expect ":KILL"
 
-# KL10 frontend directory
```
Comment (atsampson) [inline build/misc.tcl]: It'd be nicer to fix this in the previous commit (and maybe say "Tool" at the end as you've done in programs.md).

**Comment by larsbrinkhoff** [comment]
I tested the PDP-10 timesharing programs.  The standalone programs need a KL10.  I'll probably check one or two PDP-11 programs to see they're not wildly wrong, but they'll probably complain about missing KL10 gadgets.

- KLFEDR WRITE complains about .KLFE.; KLDCP (DIR).  I don't know how to create a front end filesystem or how to connect .KLFE. to it from ITS.
- FEDUMP is a standalone program, but I expect it to only run on a KL10.
- 11STNK works for linking KLRUG, KLDCP, and IOELEV.
- KLRUG and 11DDT could probably be run in 11SIM or SIMH.
- KLDCP, IO-11, and TV-11 might run partially in a simulator.
- NTSDDT, again needs a KL10.
- 11BOOT does run and creates .; @ BOOT11 as advertised.
- STUFF valrets due to a failing T11MP call.  This is expected.

File: build/misc.tcl (hunk 1 above)
Comment (larsbrinkhoff) [reply build/misc.tcl]: Thanks, I must have added this to the wrong commit.

**Comment by larsbrinkhoff** [comment]
I suppose there should be some installation instructions with DEC's original KLDCP, including how to make a front end filesystem.

**Comment by larsbrinkhoff** [comment]
KLDCP loaded into 11SIM says:

```
⊕δⓧDECSYSTEM10 DIAGNOSTIC CONSOLE⊕δⓧVERSION 0.7, MIT VERSION 20.⊕δⓧ11 SWR = 000
```

Note: I used the 11/20 simulator and 28K core.

**Comment by larsbrinkhoff** [comment]
11DDT 16K run from :PDP45 displays a `*` prompt but crashes on console input.

**Comment by larsbrinkhoff** [comment]
The KLRUG starting address is O.XXX, but there's no instruction there.  A comment says there's some magic frob for 11BOOT involved.

Starting at O.ODT seems to work.  Using PDP45 with 28K core.  There's a `*` prompt and typing `100/` displays `HALT`.

**Pull request merged**